### ISSUE TITLE
Total Reply 5.0.1: Death Sword won't load. Only shows "GLU" text on a black screen.

### ISSUE THREAD

**Issue by JDW1** [a2-4am/4cade#556]
I am using Total Reply 5.0.1 on a FloppyEMU attached to a real Apple IIc with ROM Xc installed and Plamen's VGA adapter attached to a Dell 17" LCD display.  Everything works great with this setup except the game Death Sword.  Every single time I try to load the game by pressing Return at the main splash screen, after a few seconds I see something appear that says MAXX in graphic form for a split second (200ms?) and then I get a black screen with "GLU" in white text in the middle of the screen (40-column text).  Pressing Return or joystick buttons do nothing.  My joystick is a Mach III.

Same problem happened on the slightly older v5.0 of Total Replay too.

**Comment by peterferrie** [comment]
does the stand-alone version do the same thing?
https://github.com/a2-4am/4cade/blob/main/res/dsk/Deathsword%20(4am%20crack)(4cade%20compatible).po

**Comment by JDW1** [comment]
The standalone version I use on my FloppyEMU is this WOZ disk, which works fine:

https://mirrors.apple2.org.za/ftp.apple.asimov.net/images/games/collections/woz-a-day/Death%20Sword%20%28woz-a-day%20collection%29.zip

Also note that if I load Total Replay 5.0.1 within the Virtual ][ emulator, emulating a 32K ROM Apple IIc, when I choose Death Sword, I get a black screen with white 40 column text which says:

System OK

But it freezes at that point.

Loading the WOZ disk edition of the game in Virtual ][ also works great.

So basically, it's the Death Sword version on Total Reply 5.0 and 5.0.1 that yields the black screen with white text, never loading the game.

**Comment by peterferrie** [comment]
Okay, but please still tell me how the .po version behaves for you, since it's the half-way point between the WOZ and the TR version.  It will help me know where to start looking for the issue.  It might be that the .po version has the problem, and not TR itself.

**Comment by JDW1** [comment]
Per your request, I just now downloaded the "Deathsword (4am crack)(4cade compatible).po" via the URL you previously provided to me.  The first test I always do is in the Virtual ][ emulator in my modern Mac, using an Apple IIc with 32K ROM, which matches my real hardware IIc.  When I try to boot from your *.po within Virtual ][, it shows me this...

<img width="825" alt="image" src="https://github.com/a2-4am/4cade/assets/3928027/b8ef9851-eea5-4161-ac30-dab839f31100">

I then saved the *.po onto the SD card of my FloppyEMU and tested on my real IIc.  I get the same exact error.

**Comment by JDW1** [comment]
Peter, as I just explained to you in detail in the closing paragraph at the bottom of my previous message, the game works fine when I do precisely that.

**Comment by peterferrie** [comment]
I'm sorry.  I missed that part.  Okay, I've found the issue.  The game is not compatible right now with the IIc hard-disk ROM.  The game assumes it's running on a GS and writes to a GS-specific softswitch which happens to bank out the IIc hard-disk code right before making a disk call.  That's where the crash happens.  I will make a patch for it.

**Comment by peterferrie** [comment]
fixed in d51aca547ac6a22398d5b2f7f637b3f57235179b
